docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -142,26 +151,46 @@ public class TestPluginLoader implements PluginLoader {
 Currently, you are able to add two different library types, ``JarLibrary``, and ``MavenLibraryResolver``.
 
 
-## Differences 
+## Differences
 
 ### Bukkit Serialization System
-Paper plugins do not support the serialization system (``org.bukkit.configuration.serialization``) that Bukkit uses. Your classes will not be able to be
-serialized, vise versa. It is highly recommended to not use this system with Paper plugins.
+Paper plugins still support the serialization system (``org.bukkit.configuration.serialization``) that Bukkit uses. However, custom classes will not be
+automatically registered for serialization. In order to use `ConfigurationSection#getObject`, 
+you must call `ConfigurationSerialization.registerClass(Class)` before you attempt to fetch objects from configurations.
 
 ### Classloading Isolation
-Paper plugins have isolated classloaders, meaning that relocating dependencies will not be necessary. Paper plugins are not able to access each other unless given explicit access
-by depending on another plugin, etc. This prevents plugins from accidentally accessing your dependencies, and in general helps ensure that plugins are only able to access your plugin
-if they explicitly say that they're depending on it.
+Paper plugins have isolated classloaders, meaning that relocating dependencies will not be necessary.
+Paper plugins are also not able to access each other unless given explicit access by depending on another plugin.
+This prevents plugins from accidentally accessing your dependencies, and in general helps minimise issues.
 
-Paper plugins have the ability to bypass this, being able to access OTHER plugins classloaders by adding 
+You can declare this visibility inside the [dependencies](#dependency-declaration) section:
 ```yml
-has-open-classloader: true
+Plugin:
+  join-classpath: true # Means you have access to their classpath
 ```
-to your ``paper-plugin.yml``. Note, other plugins will still be unable to access your classloader.
-
 
 ### Load Order Logic Split
 In order to better take advantage of classloading isolation, Paper plugins do **not** use the ``dependencies`` field to determine load order.
 This was done for a variety of reasons, mostly to allow better control and allow plugins to properly share classloaders.
 
-See information on [declaring load order](#load-order-declaration) for more information on how to declare the load order of your plugin.
+See information on [declaring dependencies](#dependency-declaration) for more information on how to declare the load order of your plugin.
```

Comment:
See ~~information on~~ [declaring dependencies](#dependency-declaration) for more information on how to declare the load order of your plugin.